packages/neso/package.py

```diff
@@ -69,6 +69,8 @@ def cmake_args(self):
             self.define("ENABLE_NESO_TESTS", self.run_tests),
             self.define_from_variant("ENABLE_COVERAGE", "coverage"),
         ]
+        # The line below is a work around for https://github.com/spack/spack/issues/29447
+        args.append("-DENABLE_NESO_TESTS:BOOL=ON") # TODO: Fix, issue NESO#128
```

Comment:
Better way to do this is just to remove any explicit setting of ENABLE_NESO_TESTS, as CMake has been set to default to that being true. If you'd prefer to set it explicitly here, just to be safe, you should remove line 69.

Reply:
I've made these changes now.